tsconfig.base.json

```diff
@@ -0,0 +1,17 @@
+{
+  "description": "The base configuration for all packages.",
+  "extends": "@vercel/style-guide/typescript",
+  "compilerOptions": {
```

Comment:
How about adding `outDir: "./dist"` since it looks like all of them use it?

Reply:
TypeScript is funny with relative paths. If I put `outDir` in the base config, it will output all `.js` files into a top-level `dist` directory and it doesn't take long before the build fails because a dependent package built files can't be found. We need to leave `outDir` in each package's `tsconfig.json`.

Reply:
I didn't want to touch anything having to do with paths (e.g. `include`, `exclude`, `outDir`) because it never works in monorepos. All of those relative paths are being resolved relative to the `tsconfig` file and that's not what we want.

Reply:
Another great point